### DISCUSSION TITLE
Questions and Issues Regarding Local Deployment of Supabase

### DISCUSSION THREAD

**Discussion by huiali** [supabase/supabase#26950] [Feature Requests]
Hello Supabase Team,

I have been researching the local version of Supabase for the past two weeks, and I am very passionate about this project. However, I have encountered several issues and have some questions that I hope you can help clarify:

### 1、Single Organization and Project Limitation:
I found that the local version only allows for one organization and one project. Is there a way to enable support for multiple organizations and projects, or would this require significant customization on my part? Is there an existing platform version that supports this?

### 2、 Cloud Functions Support:
The local version does not support cloud functions. I tried changing the variable `NEXT_PUBLIC_IS_PLATFORM=true` to true, which led me to the login interface, but logging in results in an error. Is there a recommended way to enable cloud functions in the local version?

### 3、Support for Multiple Databases:
Does Supabase support multiple databases, such as sharing multiple instances across `multiple` projects? Is this managed automatically, or does it require manual configuration?

### 4、Authentication Integration with `Casdoor`:
I am using `Casdoor` for `authentication` in my local setup. How can I integrate Casdoor with Supabase, or is there a way to support custom authentication methods?

I would greatly appreciate any guidance or solutions you can provide regarding these issues. Thank you for your time and support.

Best regards,

**Comment by ayushpathak-nc** [comment]
Even i am trying to set NEXT_PUBLIC_IS_PLATFORM=true. Did you get any solution to run this ?
